dev/sbt-plugin/src/main/scala/com/lightbend/lagom/sbt/InternalConfigs.scala

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright (C) 2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+package com.lightbend.lagom.sbt
+
+import sbt._
+
+object InternalConfigs {
+  private[sbt] val devModeConfig = config("lagom-dev").hide
+  private[sbt] val cassandraDevModeConfig = config("lagom-dev-cassandra").hide
```

Comment:
I wonder if, rather than manually adding these dependencies to the compile dependencies for the app when building the classpath, it would be more idiomatic sbt to make the lagom-dev configuration depend on the runtime configuration.

Some advantages to this:

* This lets ivy do the dependency resolution, if there are discrepancies between what the `lagom-dev` config has in it, and the `runtime` config has in it, currently it will just be a fluke of classpath ordering as to what will end up being used, whereas if you let ivy do the work, ivy will work it out.  This also means the end user can control somewhat how the dependencies are resolved, using ivy exclusions etc.
* It means the classpath used by Lagom in dev mode can be inspected in the ivy reports, which means when users do have a problem, we can point them to how to diagnose it.  (you may need to unhide the configuration to get that to work).

Also I'm not sure why we need a separate `lagam-dev-cassandra` config, we can already dynamically control what goes into these configurations, there doesn't seem to be a need to introduce a whole new configuration just for cassandra.